buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -154,22 +159,23 @@ interface FladleConfig {
    * these details can add additional context such as a link to the corresponding pull request.
    */
   @get:Input
-  var clientDetails: Map<String, String>
+  val clientDetails: MapProperty<String, String>
 
   /**
    * Always run - these tests are inserted at the beginning of every shard
    * useful if you need to grant permissions or login before other tests run
    */
   @get:Input
-  var testTargetsAlwaysRun: List<String>
+  val testTargetsAlwaysRun: ListProperty<String>
 
   /**
    * A list of device-path: file-path pairs that indicate the device paths to push files to the device before starting tests, and the paths of files to push.
    * Device paths must be under absolute, whitelisted paths (${EXTERNAL_STORAGE}, or ${ANDROID_DATA}/local/tmp).
    * Source file paths may be in the local filesystem or in Google Cloud Storage (gs://…).
    */
   @get:Input
-  var otherFiles: Map<String, String>
+  @get:Optional
```

Comment:
why did we add optional here, but not in other properties?

Reply:
Nice catch will remove that